tests/ml/pytorch/test_pytorch_model_upload_pytest.py

```diff
@@ -103,17 +103,31 @@ def download_model_and_start_deployment(tmp_dir, quantize, model_id, task):
 
 
 class TestPytorchModel:
+    def __init__(self):
+        # quantization does not work on Mac with ARM processor
+        import platform
+
+        self.quantize = (
+            True
+            if not (platform.system() == "Darwin" and platform.machine() == "arm64")
```

Comment:
Is it just ARM on Mac? Or would ARM on Linux be the same?

(If you don't know, you could hire a `c6g` instance in AWS to find out.)

Reply:
You are right, I had a Linux docker container for arm64 platform and it didn't support oneDNN either. Things seem to get better in Pytorch 2.0 so.